src/components/Api.js

```diff
@@ -0,0 +1,112 @@
+class Api {
+  constructor(options) {
+    this._baseUrl = options.baseUrl;
+    this._auth = options.authorization;
+    this._id;
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: {
+        authorization: `${this._auth}`,
+      },
+    }).then(this._checkResponse);
+  }
+
+  getUserData() {
+    return fetch(`${this._baseUrl}/users/me`, {
+      method: "GET",
+      headers: {
+        authorization: `${this._auth}`,
+        "Content-Type": "application/json",
+      },
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)